### ISSUE TITLE
Mothur_aligndb loc file should be divided

### ISSUE THREAD

**Issue by arbernard** [galaxyproject/tools-iuc#1383]
After testing the latest mothur wrappers, I feel like the mothur_aligndb loc file should be divided.

At the moment this file can both reference files that are alignments (used as reference for the align.seqs tool for example) and simple fasta files (used as reference for the classify.seqs tool for example).

IMHO, this situation is an issue for the end users since it might lead them in selecting improper/incompatible references in their analysis.

(ping @shiltemann)

**Comment by shiltemann** [comment]
thanks @arbernard, I agree completely, will work on this after GCC (or perhaps during hackathon)

**Comment by bgruening** [comment]
Two loc files are probably a good idea, but in theory you could also add an additional column and filter by this ... e.g. a fasta column.

**Comment by shiltemann** [comment]
good idea @bgruening, some tools can take both aligned or unaligned files as input, this may very well be true for these reference data as well for some of the tools, this approach would keep that flexibility at least

**Comment by bernt-matthias** [comment]
I guess the easiest way would be to add a column to the end (then we have backward compatibility).
